mmv1/products/cloudfunctions2/api.yaml

```diff
@@ -0,0 +1,315 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+--- !ruby/object:Api::Product
+name: Cloudfunctions2
+display_name: Cloud Functions (2nd gen)
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://cloudfunctions.googleapis.com/v2alpha/
```

Comment:
Is there a new v2 endpoint now that it's launched? Is alpha the one we want to go with?

Reply:
We're still using v2alpha post-launch. Once we reach GA I believe our endpoint will be updated.